### ISSUE TITLE
Unexpected error when converting a list of stabilizers in a state using `plaquette.pauli`

### ISSUE THREAD

**Issue by mattsantand** [qc-design/plaquette#14]
**Describe the bug**
The call `plaquette.pauli.string_to_state(["XX", "ZZ"])` throws the following error
<details>
<summary>Error traceback</summary>

```
Traceback (most recent call last):
  File "....\file.py", line 7, in <module>
    state = pauli.string_to_state(stabilisers=["XX", "ZZ"], destabilisers=["XI", "IX"])
  File "....\venv\lib\site-packages\plaquette\pauli.py", line 1152, in string_to_state
    raise ValueError(
ValueError: Destabiliser 0 does not anticommute with its stabiliser
```

</details>

**Minimal Working Example**
```
from plaquette import pauli

state = pauli.string_to_state(stabilisers=["XX", "ZZ"])
```

**Expected behavior**
I expect not to see any error. (See additional context below)

**Plaquette version:**
0.0.1a2

Commit hash: 9d5f9476c1f901326fddc9f42a760cdcc67e915b

**Comment by mattsantand** [comment]
With some digging, I noticed that the culprit might be the lines https://github.com/qc-design/plaquette/blob/9d5f9476c1f901326fddc9f42a760cdcc67e915b/src/plaquette/pauli.py#LL1123C9-L1133C10
since it calculates the destabilizers of the provided stabilizer set as a row of zeroes (identity)

**Comment by massarom** [comment]
This was fixed by #40
